front/lib/api/assistant/configuration.ts

```diff
@@ -518,34 +489,9 @@ async function fetchWorkspaceAgentConfigurationsForView(
       ),
     };
 
-    agentConfigurationType.userListStatus = agentUserListStatus({
-      agentConfiguration: agentConfigurationType,
-      listStatusOverride:
-        agentUserRelations[agent.sId]?.listStatusOverride ?? null,
-    });
-
     agentConfigurationTypes.push(agentConfigurationType);
   }
 
-  if (agentsGetView === "list") {
-    agentConfigurationTypes = agentConfigurationTypes.filter((a) => {
-      return a.userListStatus === "in-list";
-    });
-  }
-
-  if (typeof agentsGetView === "object" && "conversationId" in agentsGetView) {
```

Comment:
This one seems very specific to getting the assistant in the suggestion if we're querying from a conversation. This would break it no?

Reply:
Roger that